lib/sqs_pusher.rb

```diff
@@ -0,0 +1,19 @@
+require 'aws-sdk'
+
+
+class SqsPusher
+
+  def self.push(params)
+
+    sqs_client = Aws::SQS::Client.new
+    queue_name = 'post_test'
+    queue_url = sqs_client.get_queue_url({queue_name:queue_name}).queue_url
+
+    sqs_client.send_message({
+       queue_url: queue_url, # required
+       message_body: params.to_json, # required
+    })
+
+  end
+
+end
```

Comment:
@Tuuleh this looks good. I've a couple of suggestions:

Perhaps wrap your classes in a  module, and try to have your methods do only one thing. For example:

```ruby
module ChampaignQueue
  class Pusher
    QUEUE_NAME = "post_test"

    # Factory method
    def self.push(params)
      new(params).push
    end

    def initialize(params)
      @params = params
    end

    private

    def push
      client.send_message(
        queue_url: queue_url,
        message_body: @params.to_json,
      )
    end

    def queue_url
      ChampaignQueue.client.get_queue_url({queue_name: QUEUE_NAME}).queue_url
    end
  end

  # Client can now be accessed elsewhere.
  def client
    @client ||= Aws::SQS::Client.new
  end
end
```